README.md

```diff
@@ -4,11 +4,36 @@
 
 A simple Julia wrapper around the [PROJ](https://proj.org/) cartographic projections library.
 
-Basic example:
+Quickstart, based on the [PROJ docs](https://proj.org/development/quickstart.html):
 
 ```julia
 using Proj4
 
+# Proj4.jl implements the CoordinateTransformations.jl API
+# A Proj4.Transformation needs the source and target coordinate reference systems
+trans = Proj4.Transformation("EPSG:4326", "+proj=utm +zone=32 +datum=WGS84")
+
+# Once created, you can call this object to transform points
+# the result will be a SVector from StaticArrays.jl
+# Here the (latitude, longitude) of Copenhagen is entered
+trans([55, 12])
+# -> SVector{2, Float64}(691875.632, 6098907.825)
+
+# Note that above the latitude is passed first, because that is the axis order that the
+# EPSG mandates. If you want to pass in (longitude, latitude) / (x, y), you can set the
+# `normalize` keyword to true. For more info see https://proj.org/faq.html#why-is-the-axis-ordering-in-proj-not-consistent
+trans = Proj4.Transformation("EPSG:4326", "+proj=utm +zone=32 +datum=WGS84", normalize=true)
```

Comment:
I think `normalize` could be a confusing name here, as there's already `LinearAlgebra.normalize()` which means something very different.

Possible alternative might be `ordering=:latlong` vs `ordering=:longlat`?  `ordering` could be `axis_order`? or if you want the bool, `reorder_axes=true` perhaps?

Reply:
Yeah good point. I would prefer a boolean option. Actually I checked what pyproj uses, and I quite like their `always_xy`, 1b41e863186d4277cfe172343926f6e77c29c6e9. How's that?

Reply:
Sure, seems ok to me.